message/routes/messages.routes.js

```diff
@@ -5,36 +5,70 @@ const checkIfEmpty = require("../utils/checkIfEmpty")
 const ApiService = require('../service/api.service')
 const messageService = new ApiService()
 
+const DBService = require('../service/db.service')
+const messageDBService = new DBService()
+
+const messageDB = require('./../models/Message.model')
+
 router.get("/", (req, res) => {
-    res.json("Hello world!")
+    messageDB
+        .find()
+        .then(data => res.json(data))
+        .catch(err => res.json(err))
 })
 
 router.post("/", (req, res, next) => {
 
-    const { destination, message } = req.body
+    const { destination, message, number } = req.body
 
-    if (destination && message) {
+    if (destination && message && number) {
```

Comment:
Evaluating this (null && null && null) what will result to?

Reply:
If any of those three (or the three) enters as null, it won't enter in the if loop, and it will jump to the next if, that will display the appropiate error